snippets/mongocompat/mongotypes.js

```diff
@@ -648,7 +653,7 @@ tojsonObject = function(x, indent, nolint, depth) {
         if (typeof DBCollection != 'undefined' && val == DBCollection.prototype)
             continue;
 
-        fieldStrings.push(indent + "\"" + k + "\" : " + tojson(val, indent, nolint, depth + 1));
+        fieldStrings.push(indent + k + ": " + tojson(val, indent, nolint, depth + 1));
```

Comment:
This breaks if `k` contains non-identifier characters, right? Just checking, this is already a bit broken either way (`k` couild probably also contain double quotes or backslashes)

Reply:
Yes, it was more a change for my personal needs. 
I think I reverted it in https://github.com/mongodb-labs/mongosh-snippets/pull/9/commits/7c83f9889abe8e4ecede1b3eac6836c1a657fed8 - I am not the expert yet with git.